_posts/2016-06-15-git-merge-conflict-strategies.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "Tools and strategies for resolving Git merge conflicts"
```

Comment:
#nitpicky - I'd prefer 

> Strategies and tools for resolving Git merge conflicts

Strategies seems like the main thing, and is in the URL. Take it or leave it.